sticky_notes.py

```diff
@@ -81,3 +81,37 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_note(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) < 2:
+            return "Missing note number argument."
+
+        if len(args) >= 2:
+            nick = args[0]
+            note_num = args[1]
+
+            if nick in self.notes:
+                notes = self.notes[nick]
+                print len(self.notes[nick])
+                to_pop = -2
```

Comment:
@JakubNvk The biggest problem is that if I understand this correctly and I execute something like `del_note john doe` and we assume that there are at least two sticky notes for  `john` then this line will cause the first note to be popped (since `a.pop(0)` == `a.pop(-2)` if `len(a) == 2`).